common/shared.c

```diff
@@ -182,7 +182,9 @@ int wait_for_data_completion(struct fid_cq *cq, int num_completions)
 		if (ret > 0) {
 			num_completions--;
 		} else if (ret < 0) {
-			if (ret == -FI_EAVAIL) {
+			if (ret == -FI_EAGAIN)
+				continue;
+			else if (ret == -FI_EAVAIL) {
```

Comment:
To follow the coding guidelines, if either portion of an if statement uses braces, both portions should.  I.e. rewrite as

if (ret == -FI_EAGAIN) {
   continue;
} else ...

Problem repeated throughout patch.